src/amo/pages/Home/index.js

```diff
@@ -228,8 +228,12 @@ export class HomeBase extends React.Component {
       return null;
     };
 
+    const showHero =
+      _config.get('enableFeatureHeroRecommendation') &&
+      clientApp !== CLIENT_APP_ANDROID;
+
     return (
-      <Page isHomePage>
+      <Page isHomePage showWrongPlatformWarning={!showHero}>
```

Comment:
Hmm, actually, is this case a little different? I think not showing the hero is:

```js
!_config.get('enableFeatureHeroRecommendation') ||
clientApp === CLIENT_APP_ANDROID
```

Maybe it's the same, heh. It's hurting my brain. I suggest trying to set some shared vars to keep everything in sync.